3.analyze_data/utils/significance_testing.py

```diff
@@ -44,7 +43,35 @@ def store_comparisons(_comp_functions, _treatments, _dmso_probs, _treatment_prob
 
     return _treatments
 
-def get_treatment_comparison(_comp_functions, _treatment_paths, _probadf):
+def filter_wells_by_cell_count(_df, _cutoff):
+    """
+    Parameters
+    ----------
+    _df: pandas Dataframe
+        The dataframe which contains the well, which will be potentially removed
+
+    _cutoff: Integer
+        The number of cells a well must have to remain in the dataframe
+
+    Returns
+    -------
+    _df: pandas dataframe
+        The filtered dataframe with only wells that have more cells than the cutoff
+    """
+
+    # Find the number of cells in each well
+    well_counts = _df["Metadata_Well"].value_counts()
+
+    # Find the wells that contain more cells than the cutoff
+    wells_kept = well_counts[well_counts > _cutoff].index
```

Comment:
```suggestion
    wells_kept_index = well_counts[well_counts > _cutoff].index
```
Consider changing this var name to better reflect what the var represents

Reply:
The index in this case is just how the wells are stored